src/components/Home/BottomBanner/BottomBanner.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import Grid from 'semantic-ui-react/dist/commonjs/collections/Grid/Grid'
+
+import { DAO_DISCORD_URL, FORUM_URL, NEWSLETTER_URL } from '../../../constants'
+import { ProposalStatus, ProposalType } from '../../../entities/Proposal/types'
+import useAbbreviatedNumber from '../../../hooks/useAbbreviatedNumber'
+import useDclData from '../../../hooks/useDclData'
+import useProposals from '../../../hooks/useProposals'
+import Discord from '../../Icon/Discord'
+import EmailFilled from '../../Icon/EmailFilled'
+import Megaphone from '../../Icon/Megaphone'
+
+import Action, { ActionProps } from './Action'
+import './BottomBanner.css'
+import Stat from './Stat'
+
+const ACTIONS: Record<string, ActionProps> = {
+  discord: {
+    icon: <Discord />,
+    title: 'page.home.bottom_banner.discord_title',
+    description: 'page.home.bottom_banner.discord_description',
+    url: DAO_DISCORD_URL,
+  },
+  forum: {
+    icon: <Megaphone />,
+    title: 'page.home.bottom_banner.forum_title',
+    description: 'page.home.bottom_banner.forum_description',
+    url: FORUM_URL,
+  },
+  newsletter: {
+    icon: <EmailFilled />,
+    title: 'page.home.bottom_banner.newsletter_title',
+    description: 'page.home.bottom_banner.newsletter_description',
+    url: NEWSLETTER_URL,
+  },
+}
+
+function BottomBanner() {
+  const t = useFormatMessage()
+  const [data] = useDclData()
+  const totalFunding = useAbbreviatedNumber(data ? Number(data.funding.total) : 0)
```

Comment:
I think you may need to update this or your other PR as you modified this hook 😛

Reply:
Yep... haha